content/general/limits.textile

```diff
@@ -121,6 +122,40 @@ Bandwidth is the amount of data transferred through "messages":/glossary#message
 
 If the per-second bandwidth rate is exceeded, a "global rate limit":#global-rates will be applied to message delivery and publishing. The hourly and total bandwidth limits are "time-based":#time. If the hard limit is hit for those, message delivery and publishing will be blocked until the following hour or month.
 
+h2(#app-limits). App limits
+
+App limits relate to the quantity of resources that can be created per account.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
+| "Number of apps":#apps (per account) |\4=. 100 |
+| "Number of API keys":#api-keys (per account) |\4=. 100 |
+| "Number of rules":#rules (per account) |\4=. 100 |
+| "Number of namespaces":#namespaces (per account) |\4=. 100 |
+
+h3(#apps). Number of apps
+
+The number of apps is the maximum number of "applications":/glossary#apps that can be created per account.
+
+The number of apps per account is a "quantity-based limit":#quantity. If the limit is reached then no additional apps can be created until some of the existing ones have been deleted. Note that deleting an app will permanently delete its message history and statistics, as well as revoke access to Ably for any API keys associated with it.
+
+h3(#api-keys). Number of API keys
+
+The number of API keys is the maximum number of "API keys":/glossary#api-keys that can be created per account.
+
+The number of API keys per account is a "quantity-based limit":#quantity. If the limit is reached then no additional API keys can be created until some of the existing ones have been revoked.
+
+h3(#rules). Number of rules
+
+The number of rules is the maximum number of integration rules, such as those for "Webhooks":/general/events#configure and "Zapier":/general/events/zapier that can be created per account.
```

Comment:
I wonder if, rather than single out specific rules (and therefore leave out other ones), it would be better to just link to https://ably.com/integrations? WDYT?

 ```suggestion
The number of rules is the maximum number of "integration rules":https://ably.com/integrations that can be created per account.
```

Reply:
I linked to the specific ones simply because the parent page was linking outside of the docs site (even if it links back in), but happy to accept the change as I agree it avoids leaving one out.